boards/stm32mp157c-dk2/include/periph_conf.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2020 Savoir-faire Linux
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_stm32mp157c-dk2
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the STM32MP157C-DK2 board
+ *
+ * @author      Gilles DOFFE <gilles.doffe@savoirfairelinux.com>
+ *
+ * @}
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+#include "cfg_timer_tim2.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Clock settings
+ *
+ * @note    This is auto-generated from
+ *          `cpu/stm32_common/dist/clk_conf/clk_conf.c`
+ * @{
+ */
+/* give the target core clock (HCLK) frequency [in Hz],
+ * maximum: 209MHz */
+#define CLOCK_CORECLOCK     (208000000U)
```

Comment:
The comment says 209MHz but the define is 208MHz. Also you can use the `MHZ()` macro:
```suggestion
#define CLOCK_CORECLOCK     MHZ(208)
```
(and also for CLOCK_HSE)